packages/design-system/src/styles/_set-for-deprecation.scss

```diff
@@ -1,13 +1,72 @@
 // Classes included in this file are set for deprecation and will be removed in the next major release.
 
-// Include the soon-to-be deprecated classes here (remove them from their original files).
-// This makes it easier to find all the soon-to-be deprecated classes in one place and delete them.
-// For each class, include a comment above with the date the class was added to this file. This will help maintainers know how long a class has been listed as deprecated and when it should be removed from the project.
+// Imported to access $breakpoints to generate utility classes.
+// Remove with semantic font-size classes
+@use './layout' as *; // 💀
+$breakpoints: (
+  sm: $media-width-sm,
+  md: $media-width-md,
+  lg: $media-width-lg,
+  xl: $media-width-xl,
+) !default; // 💀
 
 // This adds a border to all soon-to-be deprecated classes.
 // When the class is deprecated, remove it from the list below.
+
+@mixin debug {
```

Comment:
Should this mixin be named `deprecate` or something?

Reply:
yeah - updating now